vars/parseStageInfo.groovy

```diff
@@ -262,28 +281,22 @@ void call(Map config = [:]) {
       tag = tag.trim()
     }
 
-    // Highest repeat ftest argument priority is TestRepeat parameter
-    if (params.TestRepeat && params.TestRepeat != '') {
-      ftest_arg_repeat = params.TestRepeat
-    } else {
-      // Next highest priority is a stage specific Test-repeat-* then the general Test-repeat
-      String repeat = cachedCommitPragma(
-        'Test-repeat' + result['pragma_suffix'], cachedCommitPragma('Test-repeat', null))
-      if (repeat) {
-        ftest_arg_repeat = repeat
-      }
+    // Get the ftest --nvme argument from either the build parameters or commit pragmas
+    String ftest_arg = get_ftest_arg(params.TestNvme, 'Test-nvme', result['pragma_suffix'])
+    if (ftest_arg) {
+      ftest_arg_nvme = ftest_arg
```

Comment:
Thanks Brian, we would also want to retain the current `ftest_arg_nvme` value if there is no parameter or pragma value being used to override it:
```suggestion
    // Get the ftest --nvme argument from either the build parameters or commit pragmas
    ftest_arg_nvme = params.TestNvme ?: cachedCommitPragma(
        'Test-nvme'+ result['pragma_suffix'], cachedCommitPragma('Test-nvme', ftest_arg_nvme))
```

We could support this by providing an optional default to `get_ftest_arg()` if we keep this function.

Reply:
> Thanks Brian, we would also want to retain the current `ftest_arg_nvme` value if there is no parameter or pragma value being used to override it:

Ahhh.  Yes, I did miss that it does already have a value when all of this is being evaulated.
 
> We could support this by providing an optional default to `get_ftest_arg()` if we keep this function.

We could, but as your suggestion notes, since `cachedCommitPragma()` already has a notion of a default value set, simply using it as you have suggested seems sufficient.